integration_tests/compute_runtime/main_test.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2024 Fastly, Inc.
+package main
+
+import (
+  "testing"
+  "time"
+
+  "github.com/fastly/compute-sdk-go/compute"
+)
+
+func TestGetVcpuMs(t *testing.T) {
+  start, err := compute.GetVCPUTime()
+  if err != nil {
+    t.Errorf("Couldn't get starting vcpu time")
+  }
+
+  time.Sleep(5 * time.Second)
```

Comment:
This is a really long sleep for tests, especially tests that don't run in parallel. I'd suggest something much shorter, and if you need to burn vCPU time, do some guest code work for a bit.

Reply:
It's more a question of trying to reduce possible flakes. Having big delays with big difference checks really reduces the possibility that we'll start seeing CI do some exciting things. In 839b205 I've updated the test down to 2 seconds, and then making sure that we haven't spent half a second in the sleep. I can go further, but my lack of knowledge of the Go runtime is making me worry that a random GC might cause a flake if I take the test too low.

Reply:
Go GC pauses are microseconds, in general, but wasi is a new world, so it is hard to know. Do the tests run on big Go, TinyGo, both?

Reply:
They should run on anything that runs on Viceroy. And I will admit I may be overly concerned about flakes, I just want to make sure the people who might see the flakes in the future are aware of the possibility. (So I'm happy to dial this down to a second sleep and a check that we haven't use 200 milliseconds. After that I worry about diminishing returns.)

Reply:
5x faster for a single test is a nice speed up! Flakes wasting people’s time are also bad, so I agree wholeheartedly to striking an appropriate balance.

Reply:
Done in d203ed1!